english/smores

```diff
@@ -0,0 +1 @@
+You decide to use your lighter and the marshmallows to make smores! It's too bad you have chocolate...
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.